articles/connections/passwordless/_setup-email.md

```diff
@@ -8,7 +8,29 @@ In the Dashboard, on the **Email** page, under [Connections > Passwordless](${ui
 
 ![](/media/articles/connections/passwordless/passwordless-email-config.png)
 
-The email can be written in HTML with Liquid syntax, allowing you to conditionally shape its content. The following macros are available when defining the template:
+##### Multi-Language Support
 
- - `{{ application.name }}`
- - `{{ code }}` (the one-time code)
+The Message area supports usage of multiple languages.
+
+By making a call to the [/passwordless/start](/api/authentication#!#post--with_email) authentication endpoint, you can set the value of an 'x-request-language' header to the language of your choice. If the value of this header is not set, the language will be extracted from the value in the 'accept-language' header that is automatically set by the browser.
+
+The Message area accepts Liquid syntax. You can use this syntax, combined with exposed parameter values, to programmatically construct elements of the message. For example, you can reference the `request_language` parameter to change the language of the message:
+
+```text
+{% if request_language contains 'dutch' %}
+   Hier is uw verificatie code: @@password@@
```

Comment:
For emails the parameter is `code` not `password`. For Liquid syntax the way to render variables is {{ xxx }}
So, it should be: `Hier is uw verificatie code: {{ code }}`